libs/api/src/main/java/de/cyface/api/MeasurementIterator.java

```diff
@@ -157,7 +157,7 @@ public void handle(final JsonObject document) {
             // Handle bucket of next measurement (i.e. all buckets of previous measurement loaded)
             // System.out.println("MeasurementStream.handle(next measurements buckets) -> resolveNext");
             resolveNext(cachedBuckets, measurementId, bucket);
-        } catch (ParseException e) {
+        } catch (ParseException | RuntimeException e) {
```

Comment:
Without this the provider API did not respond when it received a request to load data in an unsupported version as the Runtime Exception was not passed to the failure handler.

Not it responds with 500, which is okay.